src/octokit_utils/index.ts

```diff
@@ -1,127 +1,185 @@
 
-function initPullRelatedRequest(context: Context<Webhooks.WebhookPayloadPullRequest>): any {
-  const pullNumber = context.payload.pull_request.number
-  const repo = context.payload.repository.name
-  const owner = context.payload.repository.owner.login
-  context.log.info(`Initializing pull related request with ${owner}/${repo} #${pullNumber}`)
-  return { pull_number: pullNumber, owner, repo }
-}
+const initPullRelatedRequest = function (
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+): any {
+  const pullNumber = context.payload.pull_request.number;
+  const repo = context.payload.repository.name;
+  const owner = context.payload.repository.owner.login;
+  context.log.info(
+    `Initializing pull related request with ${owner}/${repo} #${pullNumber}`,
+  );
+  let req: any = { owner, repo };
+  req["pull_number"] = pullNumber;
+  return req;
+};
 
-async function approveChange(context: Context<Webhooks.WebhookPayloadPullRequest>): Promise<void> {
-  const req = initPullRelatedRequest(context)
-  req.event = 'APPROVE'
-  context.log.info(`Reviewing PR with request ${JSON.stringify(req)}`)
+const approveChange = async (
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+): Promise<void> => {
+  const req = initPullRelatedRequest(context);
+  req.event = "APPROVE";
+  context.log.info(`Reviewing PR with request ${JSON.stringify(req)}`);
   try {
-    const res = await context.github.pulls.createReview(req)
+    const res = await context.github.pulls.createReview(req);
     if (res.status === 200) {
-      context.log.info('Approve changes succeeded.')
+      context.log.info("Approve changes succeeded.");
     } else {
-      context.log.error(`Approve change rejected with: ${JSON.stringify(res.data)}`)
+      context.log.error(
+        `Approve change rejected with: ${JSON.stringify(res.data)}`,
+      );
     }
   } catch (err) {
-    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`)
+    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`);
   }
-}
+};
 
-async function createPassingStatus(context: Context<Webhooks.WebhookPayloadPullRequest>, startTime: string, endTime: string): Promise<void> {
-  const statusOptions: (Octokit.RequestOptions & Octokit.ChecksCreateParams) = context.repo({
-    name: APP_CHECK_NAME,
-    head_sha: context.payload.pull_request.head.sha,
-    status: 'completed',
-    started_at: startTime,
+const createPassingStatus = async (
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+  startTime: string,
+  endTime: string,
+): Promise<void> => {
+  const statusOptions: Octokit.RequestOptions &
+    Octokit.ChecksCreateParams = context.repo({
     completed_at: endTime,
-    conclusion: 'success',
+    conclusion: "success",
+    head_sha: context.payload.pull_request.head.sha,
+    name: APP_CHECK_NAME,
     output: {
-      title: 'test',
-      summary: 'test',
-      text: 'test',
+      summary: "test",
+      title: "test",
+      text: "test",
```

Comment:
Codacy found an issue: [The key 'text' is not sorted alphabetically](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)